docs/source/tutorials_source/platform/tutorial_active_learning_detectron2.py

```diff
@@ -0,0 +1,306 @@
+        x0, y0, x1, y1 = bbox_raw.cpu().numpy()
+        x0 /= width
+        y0 /= height
+        x1 /= width
+        y1 /= height
+      
+        boxes.append(BoundingBox(x0, y0, x1, y1))
+    output = ObjectDetectionOutput.from_scores(boxes, outputs['instances'].scores.cpu().numpy(), outputs['instances'].pred_classes.cpu().numpy().tolist())
+    return output
+
+# %%
+# Run model predictions
+# ----------------------
+#
+# We now use the created model and iterate over the `query_set` and make predictions.
+# It's important that the predictions are in the same order as the filenames
+# in the `query_set`. Otherwise, we could upload a prediction to the wrong sample!
+
+obj_detection_outputs = []
+pbar = tqdm.tqdm(al_agent.query_set)
+for fname in pbar:
+  fname_full = os.path.join(DATASET_ROOT, fname)
+  im = cv2.imread(fname_full)
+  out = predictor(im)
+  obj_detection_output = convert_bbox_detectron2lightly(out)
+  obj_detection_outputs.append(obj_detection_output)
+
+# %%
+# Now, we need to turn the predictions into scores.
+# The scorer assigns scores between 0.0 and 1.0 to 
+# each sample and for each scoring method.
+scorer = ScorerObjectDetection(obj_detection_outputs)
+scores = scorer.calculate_scores()
+# %% 
+# Let's have a look at the sample with the highest
+# prediction-margin score.
+max_score = scores['uncertainty_margin'].max()
+idx = scores['uncertainty_margin'].argmax()
+print(f'Highest uncertainty_margin score found for idx {idx}: {max_score}')
+
+# %%
+# Let's have a look at this particular image and show the model 
+# prediction for it.
+fname = os.path.join(DATASET_ROOT, al_agent.query_set[idx])
+predict_and_overlay(predictor, fname)
+
+# %%
+# Finally, we can tell our agent to select the top 100 images to annotate and
+# improve our existing model. We pick the sampling method called `CORAL` which
+# is a combination of Coreset and Active Learning. Whereas Coreset maximizes
+# the image diversity based on the embeddings, active learning aims at selecting
+# images where our models struggles the most. 
+config = SamplerConfig(
+  n_samples=100, 
+  method=SamplingMethod.CORAL, 
+  name='active-learning-loop-1'
+)
+al_agent.query(config, scorer)
+
+# %%
+# We can access the newly added data from the agent.
+print(len(al_agent.added_set))
+
+# %%
+# Let's have a look at the first 5 entries.
+print(al_agent.added_set[:5])
+
+# %%
+# Let's show model predictions for the first 5 images.
+to_label = [os.path.join(DATASET_ROOT, x) for x in al_agent.added_set]
+for i in range(5):
+  predict_and_overlay(predictor, to_label[i])
+
+# %%
+# What's Next?
+# -------------
+# 
+# We showed in this tutorial how you can use Lightly Active Learing to discover 
+# the images you should label next. You can close the loop by annotating 
+# the 100 images and re'train your model. Then start the next iteration 
+# by making new model predictions on the updated query_set (which will now 
```

Comment:
The query set doesn't change. Maybe change this sentence to:

Then start the next iteration by making new model predictions with the updated model.